packages/react-vega/src/Vega.jsx

```diff
@@ -4,12 +4,14 @@ import * as vega from 'vega';
 
 import PropTypes from 'prop-types';
 import React from 'react';
+import vegaEmbed from 'vega-embed';
 import { capitalize, isDefined, isFunction } from './util';
 
 const propTypes = {
   background: PropTypes.string,
   className: PropTypes.string,
   data: PropTypes.object,
+  embedOption: PropTypes.object,
```

Comment:
Let's not add this yet and instead just redirect the existing options as suggested in another comment.

Reply:
Just coming to say that in my opinion this was a bad decision. Disabling this escape hatch mean that as a user of the library, I have no way of easily disable the vega-embed action button, which iI don't want.